### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by lenaorobei**
## Purpose of this pull request

This PR fixes information about test instance deployer:
- Fixed info about supported tags and branches.
- Removed part about `Environment` that is not yet implemented.

## Affected DevDocs pages

-  https://devdocs.magento.com/contributor-guide/contributing.html

### CODE REVIEW

File: src/contributor-guide/contributing.md [hunk 1]
```diff
@@ -206,7 +206,7 @@ When you want to verify an issue or pull request, use the `instance` command to
 @magento give me {$version} instance
 ```
 
-Replace `{$version}` with the version tag for the version required. The currently supported values are the latest [version tags](https://github.com/magento/magento2/tags) and the 2.4-develop branch. For example:
+Replace `{$version}` with the version tag or branch. The currently supported values are: the latest released version and the `2.4-develop` branch. For example:
```
Comment (BarnyShergold) [inline src/contributor-guide/contributing.md:209]:
```suggestion
Replace `{$version}` with the version tag or branch. The currently supported values are, the latest released version and the `2.4-develop` branch. For example:
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Tiny change recommended

File: src/contributor-guide/contributing.md [hunk 2]
```diff
@@ -206,7 +206,7 @@ When you want to verify an issue or pull request, use the `instance` command to
 @magento give me {$version} instance
 ```
 
-Replace `{$version}` with the version tag for the version required. The currently supported values are the latest [version tags](https://github.com/magento/magento2/tags) and the 2.4-develop branch. For example:
+Replace `{$version}` with the version tag or branch. The currently supported values are, the latest released version and the `2.4-develop` branch. For example:
```
Comment (meker12) [inline src/contributor-guide/contributing.md:209]:
```suggestion
Replace `{$version}` with the version tag or branch. The following values are supported for `{$version}`: the version number for the latest release, and `2.4-develop` for the development branch.
```

**Comment by meker12** [comment]
running tests

**Review by meker12 — APPROVED** [review]

**Pull request merged**